packages/fxa-auth-server/lib/email/utils/helpers.js

```diff
@@ -6,7 +6,7 @@
 
 const ROOT_DIR = '../../..';
 
-const config = require(`${ROOT_DIR}/config`);
+const config = require(`${ROOT_DIR}/config`).default;
```

Comment:
nit: Personally I like this syntax more: `const { default:config } = require('../../../config')`